### PR TITLE
Integration test to verify cluster components are healthy.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by dlorenc** [comment]
This currently requires you to setup kubectl to point at your cluster. Once #9 is fixed, we can call start automatically.

File: test/integration/util/util.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package util
```
Comment (luxas) [inline test/integration/util/util.go]: Add copyright, and maybe `// +build integration` if that's required

**Comment by luxas** [comment]
This LGTM

**Comment by luxas** [comment]
Soon, we should investigate if we can run integration tests on travis too... (Or GCE e2e)

**Pull request merged**